components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -987,10 +995,10 @@ + (void)setFloatingPlaceholderScaleDefault:(CGFloat)floatingPlaceholderScaleDefa
 
 - (void)setHelperText:(NSString *)helperText {
   if (self.isDisplayingErrorText) {
-    self.previousLeadingText = helperText;
+    self.previousLeadingText = [helperText copy];
```

Comment:
Does this `copy` change need to land in this PR? Might be good to do in a separate PR.

Reply:
I can go ahead and take it out 👍